observing/schedule.py

```diff
@@ -60,6 +61,32 @@ def get_sched():
     return dd, dd2
 
 
+def check_sched(sched):
+    """ Check if sched is requesting a beam that is already scheduled or submitted
+    """
+
+    dd, dd2 = get_sched()
+    sched_dict = create_dict(sched)
+
+    ok = True
+    for kk,vv in sched_dict.items():
```

Comment:
Um...

I don't understand this section. Please consider renaming variables. Also can you do the check with schedules as dataframes and not as dicts?

Reply:
I'll rename variables and add comments. Could consider using a dataframe in the future.